core/arch/x86/decode_table.c

```diff
@@ -1266,7 +1266,7 @@ const instr_info_t * const op_instr[] =
     /* OP_vpbroadcastd  */   &third_byte_38[118],
     /* OP_vpbroadcastq  */   &third_byte_38[119],
 
-    /* added in Intel Skylake */
+    /* Added in Intel Skylake */
```

Comment:
Per style guide: punctuate as well (end in period, or maybe colon here) and ideally make it a complete sentence.  In general, either capitalized + punctuated + complete sentences, or short snippet where we don't care, with preference in new code for the former.  So it seems either leave it alone or "Opcodes added in Intel Skylake:" or sthg.

Reply:
Ok. I reverted since it is unrelated to this patch.